doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -323,6 +323,13 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-5-0 v2-4-0 v2-3-0 v2-2-0 v2-1-0 v2-0-0 v1-9-0 v1-8-0 v1-7-0
+
+KRKNWK-18495: When dimming a light/bulb over time or changing color, the thread network can be spammed.
```

Comment:
I know this is a description that origins from the ticket, but could we modify it to more clearly describe how it affects NCS (e.g. that it affects only Matter Light Bulb sample (link it), explain what actually is Thread network error).

Reply:
I have to agree that this needs additional clarification. For example, I am not sure what "over time" is doing here. The interpretations I can draw, in order of most to least likely based on current wording:

* When a light bulb is set to dim or change color over a long period of time (so a very gradual change), it can end up spamming the Thread network during that time.
* When a light bulb is dimmed or its color is changed several times over a long period of time, it can start spamming the Thread network.
* Either of the two above options, but the spamming only happens when the process is started.

Reply:
I changed the description. @kkasperczyk-no, @wiba-nordic  please take a look.